website-page1.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link href="style.css" rel="stylesheet">
+    <title>Muna Fidow Website</title>
+    <style="background-color: darkolivegreen;"></style>
```

Comment:
Style tags are used differently - they should contain the css code instead of it being an attribute (like on other html elements). Look into style tags